website/meta/universe.json

```diff
@@ -1801,6 +1801,34 @@
                 "github": "microsoft"
             }
         },
+        {
+            "id": "python-sentence-boundary-disambiguation",
+            "title": "pySBD - python Sentence Boundary Disambiguation",
+            "slogan": "a rule-based sentence boundary detection that works out-of-the-box",
+            "github": "nipunsadvilkar/pySBD",
+            "description": "pySBD is 'real-world' sentence segmenter which extracts a reasonable sentences when the format and domain of the input text are unknown. It is a rules-based algorithm based on [The Golden Rules](https://s3.amazonaws.com/tm-town-nlp-resources/golden_rules.txt) - a set of tests to check accuracy of segmenter in regards to edge case scenarios developed by [TM-Town](https://www.tm-town.com/) dev team. pySBD is python port of ruby gem [Pragmatic Segmenter](https://github.com/diasks2/pragmatic_segmenter).",
+            "pip": "pysbd",
+            "category": ["scientific"],
+            "tags": ["sentence segmentation"],
+            "code_example": [
+                "import spacy",
+                "import pysbd",
```

Comment:
You don't even need this line anymore 🙂

```suggestion
```

Although I'd probably show the straightforward construction first, using the factory directly? Makes it a bit more obvious what's going on:

```python
from pysbd.util import PySBDFactory

nlp.add_pipe(PySBDFactory(nlp))
```

Reply:
Agree! making it more readable :)